src/components/Auth/components/Login.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { withStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import Input from '@material-ui/core/Input';
+import InputLabel from '@material-ui/core/InputLabel';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import FormControl from '@material-ui/core/FormControl';
+
+import styles from './Login.styles.js';
+
+class Login extends React.Component {
+  state = {
+    email: '',
+    password: '',
+  };
+
+  handleInputChange = name => (event) => {
+    this.setState({
+      [name]: event.target.value,
+    });
+  }
+
+  handleSubmit = () => {
+    // This prevents a '?' from being appended to the URL
+    if (!this.props.waiting) {
+      this.props.handleLogin(this.state.email, this.state.password);
+    }
+  }
+
+  render() {
+    const { logErr, classes, waiting } = this.props;
+    const { email, password } = this.state;
+    return (
+      <form className={classes.container} noValidate autoComplete='off'>
+        <FormControl className={classes.text}>
+          <InputLabel htmlFor='email-input'>Email</InputLabel>
+          <Input
+            id='email-input'
+            type='email'
+            value={email}
+            onChange={this.handleInputChange('email')}
+          />
+        </FormControl>
+        <FormControl
+          className={classes.text}
+          error={Boolean(logErr)}
+        >
+          <InputLabel htmlFor='password-input'>Password</InputLabel>
+          <Input
+            id='password-input'
+            type='password'
+            value={password}
+            onChange={this.handleInputChange('password')}
+          />
+          {/* Display Error only when there is an error in the password */}
+          {/* TODO: Turn this into a snackbar */}
+          {
+            this.props.logErr
+              ? <FormHelperText error>{logErr}</FormHelperText>
+              : null
+            }
+        </FormControl>
+        <Button
+          id='login-button'
+          className={classes.button}
+          fullWidth
+          variant='contained'
+          color='primary'
+          disabled={waiting}
+          onClick={this.handleSubmit}
+        >
+          Login
+        </Button>
+      </form>
+    );
+  }
+}
+
+Login.propTypes = {
```

Comment:
Yeah, Flow wasn't working for with me for some reason. And I couldn't get flow to stop getting mad about the absolute pathing. Idk. propTypes is already in use and it works just fine. Plus we both know how to use it. Another thing. Flow adds on to our build process since it requires a babel preset to work. Thats more time spent in babel and less time spent testing and developing along with the overhead of running the flow server in the background of our computers. PropTypes doesn't need that.